.github/workflows/BuildImage.yml

```diff
@@ -6,8 +6,13 @@ jobs:
   build:
     env:
       DOCKERHUB: "linuxserver/mods" #don't modify
+<<<<<<< HEAD
       BASEIMAGE: "baseimagename" #replace
       MODNAME: "modname" #replace
+=======
+      BASEIMAGE: "lidarr" #replace
+      MODNAME: "flac2mp3" #replace
+>>>>>>> 38e21e1... Initial release
```

Comment:
These artifacts need to be removed

Reply:
Gah!  Sorry about that, very sloppy of me.  Now fixed.